automatic/seafile-client/tools/chocolateyInstall.ps1

```diff
@@ -1,15 +1,24 @@
-﻿$packageName = '{{PackageName}}'
-$fileType = 'msi'
-$silentArgs = '/passive'
-$url = '{{DownloadUrl}}'
-$version = '{{PackageVersion}}'
+﻿$ErrorActionPreference = 'Stop'
+
+$packageArgs = @{
+  packageName = 'seafile-client'
+  fileType    = 'msi'
+  softwareName = 'Seafile 6.0.0'
+
+  checksum     = '64f0e7192acbbbad4e6ac34d5b531ff27697172b9109f2c00cf61ebae923a642'
+  checksumType = 'sha256'
+  url          = 'https://bintray.com/artifact/download/seafile-org/seafile/seafile-6.0.0-en.msi'
+
+  silentArgs   = '/passive'
+  validExitCodes = @(0)
+}
 
 $app = Get-WmiObject -Class Win32_Product | Where-Object {
```

Comment:
Doesn't this ignores the version ?
Also, this wmi class is **very** slow. Perhaps it would be better to check registry key as in uninstaller ? It would also give version.

But is it really necessary ? What is the position on this anyway @gep13 ? I usually remove those checks if files are not big (less then 100MB for me) and keep them on big downloads.